tests/tests/proposal_tests.cpp

```diff
@@ -56,7 +192,7 @@ BOOST_AUTO_TEST_CASE(create_proposal) { try {
     cop.proposed_operations.push_back(operation_wrapper(top));
     cop.proposed_operations.push_back(operation_wrapper(vop));
 
-    BOOST_REQUIRE_THROW(push_tx_with_ops(tx, bob_private_key, cop), fc::exception);
+    BOOST_REQUIRE_THROW(push_tx_with_ops_throw(tx, bob_private_key, cop), fc::exception);
```

Comment:
At this point, you need to check for what error is generated. Using fc::exception does not give any guarantee that the correct error is generated. From the point of view of error checking, this is equivalent to the arrival of any exception and it is unclear what the user will see in this case. Moreover, I met the places where the test passes, but it does not at all check what the developer has laid down.